tests/api_resources/abstract/test_custom_method.py

```diff
@@ -30,87 +30,65 @@ def do_stream_stuff(self, idempotency_key=None, **params):
             return self.request_stream("post", url, params, headers)
 
         @classmethod
-        def _cls_do_stuff_new_codegen(
-            cls,
-            id,
-            api_key=None,
-            stripe_version=None,
-            stripe_account=None,
-            **params
-        ):
+        def _cls_do_stuff_new_codegen(cls, id, **params):
```

Comment:
Are we going to make this generator change as well?

Reply:
Ah I included too many changes, reverted with https://github.com/stripe/stripe-python/pull/1179/commits/f2441927ad1cfd7337ab3331f0c59014d911a77f